### PR TITLE
Minor Spelling and Grammar Corrections

### PR DESCRIPTION
**Pull request by gfyoung**
Self-explanatory PR.

### CODE REVIEW

File: numpy/random/tests/test_random.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def test_negative_binomial(self):
         self.prng.negative_binomial(0.5, 0.5)
 
 class TestRandomDist(TestCase):
-    # Make sure the random distrobution return the correct value for a
+    # Make sure the random distribution return the correct value for a
```
Comment (charris) [inline numpy/random/tests/test_random.py]: return <- returns

**Comment by gfyoung** [comment]
Ah, good catch.  Fixed and updated.

**Comment by charris** [comment]
Close and reopen to restart tests.

**Comment by gfyoung** [comment]
@charris : Thanks!  If Travis is happy, and there are no obvious grammar/spelling errors in the files I modified, this PR should be good to merge.

**Comment by charris** [comment]
LGTM, thanks @gfyoung .

**Pull request merged**